hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -514,3 +514,43 @@ def hpat_pandas_series_floordiv_number_impl(lhs, rhs):
         return hpat_pandas_series_floordiv_number_impl
 
     raise TypingError('{} The object must be a pandas.series or scalar. Given rhs: {}'.format(_func_name, rhs))
+
+
+@overload_method(SeriesType, 'quantile')
+def hpat_pandas_series_quantile(self, q=.5):
```

Comment:
it might be better to mention all parameters like in Pandas `(self, q=0.5, interpolation='linear')` and just don't use (mark as unsupported) parameter `interpolation`